### PR TITLE
Rename one letter variables

### PR DESCRIPTION
**Pull request by jacobherrington**
**Description**

This is a small tweak, personally, I find single-character variable names like these to be less than ideal. IMO no reason to not change these. 🤠 

**Checklist:**
- [x] I have followed [Pull Request guidelines](https://github.com/solidusio/solidus/blob/master/CONTRIBUTING.md#pull-request-guidelines)
- [x] I have added a detailed description into each commit message
- [ ] I have updated Guides and README accordingly to this change (if needed)
- [ ] I have added tests to cover this change (if needed)

### CODE REVIEW

**Comment by jacobherrington** [comment]
@spaghetticode There are a bunch of these throughout the codebase. I might go in and do a more sweeping change in the future, but it would also make a good first-time contribution for anyone new to open source 😉

**Comment by spaghetticode** [comment]
@jacobherrington I think that's a great idea! I will open an issue and mark it as beginner-friendly

File: backend/app/helpers/spree/admin/navigation_helper.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ def admin_page_title
         elsif content_for?(:page_title)
           content_for(:page_title)
         elsif admin_breadcrumbs.any?
-          admin_breadcrumbs.map{ |x| strip_tags(x) }.reverse.join(' - ')
+          admin_breadcrumbs.map{ |breadcrumb| strip_tags(breadcrumb) }.reverse.join(' - ')
```
Comment (kennyadsl) [inline backend/app/helpers/spree/admin/navigation_helper.rb]: What about taking advantage of the change on this line and also add a space after `map` for consistency?

Comment (jacobherrington) [reply backend/app/helpers/spree/admin/navigation_helper.rb]: Sure, good call. 👍

**Pull request merged**